janusgraph-core/src/main/java/org/janusgraph/graphdb/query/QueryUtil.java

```diff
@@ -67,7 +67,8 @@ public static InternalRelationType getType(StandardJanusGraphTx tx, String typeN
 
     public static Iterable<JanusGraphVertex> getVertices(StandardJanusGraphTx tx,
                                                     PropertyKey key, Object equalityCondition) {
-        return tx.query().has(key,Cmp.EQUAL,equalityCondition).vertices();
+        //TODO why did I have to add .name()?
+        return tx.query().has(key.name(),Cmp.EQUAL,equalityCondition).vertices();
```

Comment:
@sjudeng what is going on here, my IDE tells me it is a compilation error if I dont add .name() to key.

Reply:
seems like there was stale class data someplace in the classpath. works now